DEPRECATIONS.md

```diff
@@ -4,6 +4,7 @@ This document lists deprecated constructs in the SDK, with their expected time 
 
 | API / Feature | Deprecated (warning) | Deprecated (error) | Removed | Notes |
 | --- | --- | --- | --- | --- |
+| `GetUnreadChannelCount#invoke`<br/> | 2021.03.16<br/>4.7.1  | 2021.04.16 ⌛ | 2021.05.16 ⌛ | Use ChatDomain::channelUnreadCount instead |
```

Comment:
I think this is he version the target method/class is deprecated from. In our case, the next release is 4.7.2